rspec/testing_cors.md

```diff
@@ -0,0 +1,31 @@
+It's a bit tricky to test with `RSpec`, because we don't have a real server (like [Webrick](https://github.com/nahi/webrick), [Unicorn](http://unicorn.bogomips.org/) or [Puma](https://github.com/puma/puma)) between the code and the client. This usually is not a problem but it does affect in this particular case, and the [rfc3875](https://tools.ietf.org/html/rfc3875#section-4.1.18) explains why. The server translate the custom `HTTP` request headers in this way:
+
+* Convert to upper case.
+* Replace `-` with `_`
+* Prepend `HTTP_`.
+
+Because we don't have the server to do it for us, we must do it manually. For example, the `Origin` becomes `HTTP_ORIGIN` and `Http-Access-Control-Request-Method` becomes `HTTP_ACCESS_CONTROL_REQUEST_METHOD`. Let's see some example tests:
+
+####Sending the Origin header:
+
+```ruby
+scenario 'Returns the response CORS headers' do
```

Comment:
Technically, `scenario` isn't part of core RSpec language – in Custard, we use custom synonyms. 

Would specifying `RSpec.feature` be better, perhaps?